src/select-panel/select-item.tsx

```diff
@@ -46,8 +46,7 @@ const SelectItem = ({
 
   useEffect(() => {
     updateFocus();
-    // eslint-disable-next-line
-  }, [focused]);
+  }, [checked, focused]);
```

Comment:
please add `// eslint-disable-next-line` back since it's necessary for lint

Reply:
Sure thank you